src/main/java/com/crud/tasks/scheduler/EmailScheduler.java

```diff
@@ -20,11 +20,15 @@
     @Scheduled(fixedDelay = 10000)
     public void sendInformationEmail() {
         long size = taskRepository.count();
+        String name = "tasks";
+        if(size==1) {
+            name = "task";
+        }
```

Comment:
1. Brakuje spacji
2. Mozna to zapisac prosciej bez nadpisywania stringa, np
String name = size==1? " task" : "tasks"